invenio_app_rdm/config.py

```diff
@@ -649,7 +649,7 @@ class UserPreferencesNotificationsSchema(UserPreferencesSchema):
 }
 
 APP_RDM_ROUTES = {
-    "index": "/",
+    "index": ("/", None),
```

Comment:
by default this should be left as `/` right?

Reply:
I'm not sure what you mean?